src/rsqsim_api/rsqsim_api/catalogue/catalogue.py

```diff
@@ -127,6 +134,22 @@ def patch_slip(self, data_list: np.ndarray):
         self.check_list(data_list, data_type="d")
         self._patch_slip = data_list
 
+    @property
+    def accumulated_slip(self):
+        return self._accumulated_slip
+
+    @accumulated_slip.setter
+    def accumulated_slip(self, data_list: dict):
+        self._accumulated_slip = data_list
```

Comment:
```suggestion
```

Reply:
See other changes... I tend not to use these if I have the functions that populate the data list like you do.